src/Form.svelte

```diff
@@ -267,7 +268,7 @@
           {/if}
         </div>
       {:else}
-        <div class="description txt-faded txt-small">
+        <div class="description txt-faded txt-small" use:twemoji>
```

Comment:
Do we have emojis in any of the descriptions?

Reply:
Can be removed 👍

Reply:
b7e034ac